lib/session.js

```diff
@@ -24,6 +25,7 @@ module.exports = class Session {
     this[cookieSignerKey] = cookieSigner
     this[requestKey] = request
     this[sessionIdKey] = sessionId
+    this[encryptedSessionIdKey] = (prevSession && prevSession[sessionIdKey] === sessionId && prevSession[encryptedSessionIdKey]) || cookieSigner.sign(this.sessionId)
```

Comment:
this logic is hard to read, could you split it into some `if`s instead of 3 inline conditionals?

Reply:
but it is branchless code?!

Reply:
I reformatted it. Is it acceptable now :)?